phil/what-is-master.md

```diff
@@ -10,43 +39,54 @@ Currently, it is "skimmed-over code" (not necessarily tested).
   - Code would still go through code review **of course**
 - "I don't want to be testing every little change; I want to batch them"
   - Code reviewers could still merge "non-hairy" Pull Requests
-  - but would still need a way to deploy the PullRequest _with master merged in_ (`git fetch origin pull/${PR_NUMBER}/merge`)
 
 Here's why it shouldn't be scary:
 
-1. Accepting a ticket is a _better_ indicator for putting code in master than "the code reads well" (most of the time)
+1. Accepting a ticket is a _better_ indicator for putting code in `/master` than "the code reads well" (most of the time)
 1. QA knows exactly what is going out (and can bring CS in earlier)
   - untested code does not "sneak in"
 1. it's just as easy as clicking "Accept" on the ticket
   - BE and FE PR's can be tested and merged at the same time (automatically)
 
 
-2 options for testing:
+# Options for Testing
+
+1. devs _and_ QA should be able to easily deploy PR's (not commits) to environments
+1. we should use manifestly properly; **move** from QA -> Staging -> Production
+
+(actually, they're the same, except #1 is how manifests are created :smile:)
 
-1. be able to easily deploy PR's to environments (devs _and_ QA can test)
-1. use manifestly properly; **move** from QA -> Staging -> Production.
 
-(actually, they're the same, except #1 is more verbose)
+# Manifestly Usage
 
+It documents when changes in multiple repos are tied together (this was the cause of the production rollback. QA had a different set of commits than production).
 
+**Note:** If we use manifestly properly, then we _could_ wait to merge to `/master` until after code is released to production; see "hotfix".
```

Comment:
We _could_ but I would hesitate to do so... I don't like waiting to merge, as it gets to be a pain for developers.  But I agree with your other points above, so really for features we probably do want more feature flags to disable pieces of code.